nebula-components/components/_text-input.scss

```diff
@@ -14,4 +14,18 @@
   .#{$nb-namespace}c-text-input--textarea {
     min-height: 10rem;
   }
+
+  .#{$nb-namespace}c-text-search {
+    position: relative;
+  }
+
+  .#{$nb-namespace}c-text-input--search {
+    padding-right: ($nb-text-input-padding * 5);
+  }
+
+  .#{$nb-namespace}c-btn-icon--search {
+    background-position: center center;
+    transform: rotate(90deg);
```

Comment:
Just a small thing, we might wanna look at actually rotating the SVG itself first rather than adding the extra CSS